File: src/main/java/com/example/sk/taskmanagement/domain/CategoryTaskView.java
package com.example.sk.taskmanagement.domain;

import java.time.Instant;
import java.time.LocalDate;

public record CategoryTaskView(Long categoryId,
                               String categoryName,
                               Long taskId,
                               String taskDescription,
                               Instant taskCreationDate,
                               LocalDate taskDueDate) {
}

File: src/main/kotlin/com/example/sk/taskmanagement/domain/CategoryTaskView.kt
package com.example.sk.taskmanagement.domain

import java.time.Instant
import java.time.LocalDate

data class CategoryTaskView(
    val categoryId: Long,
    val categoryName: String,
    val taskId: Long,
    val taskDescription: String,
    val taskCreationDate: Instant,
    val taskDueDate: LocalDate,
)
